message-fanout/platform/sim.w

```diff
@@ -0,0 +1,13 @@
+bring cloud;
+bring "../commons/api.w" as api;
+
+pub class MessageFanout_sim impl api.IMessageFanout {
+  topic: cloud.Topic;
+
+  new() {
+    this.topic = new cloud.Topic();
+  }
+
+  pub addConsumer(handler: inflight(str): void, props: api.MessageFanoutProps): void {}
```

Comment:
So this does not do anything in the sim currently?

Reply:
Sorry Hasan, I had forgotten to implement it, now it's correct